src/deepsparse/transformers/helpers.py

```diff
@@ -119,19 +130,21 @@ def get_onnx_path_and_configs(
         )
         if tokenizer_config_path is not None:
             tokenizer_config_path.path  # trigger download of tokenizer_config
-    elif require_configs and (config_path is None or tokenizer_path is None):
-        raise RuntimeError(
-            f"Unable to find model and tokenizer config for model_path {model_path}. "
-            f"model_path must be a directory containing model.onnx, config.json, and "
-            f"tokenizer.json files. Found config and tokenizer paths: {config_path}, "
-            f"{tokenizer_path}"
-        )
     else:
         raise ValueError(
-            f"model_path {model_path} is not a valid file, directory, or zoo stub"
+            f"model_path {model_path} is not a valid directory or zoo stub"
+        )
+
+    if config_path is None or tokenizer_path is None:
+        warnings.warn(
+            f"Unable to find model or tokenizer configs for model_path {model_path}. "
+            f"model_path must be a directory containing config.json, and/or "
```

Comment:
"and/or"? not "and"?

Reply:
If only one is present it's fine as we no longer require a `deployment` folder with both files and can expect a config or tokenizer through one of the other now supported input types (paths, dict, etc)